deployment/ccip/changeset/v1_5_1/cs_configure_token_pools.go

```diff
@@ -161,7 +169,14 @@ func (c ConfigureTokenPoolContractsConfig) Validate(env deployment.Environment)
 				return fmt.Errorf("missing proposerMcm on %s", chain.String())
 			}
 		}
-		if err := poolUpdate.Validate(env.GetContext(), chain, chainState, c.MCMS != nil, c.TokenSymbol); err != nil {
+		tokenSymbol := c.TokenSymbol
```

Comment:
Let's move this logic into poolUpdate.Validate, we should still pass c.TokenSymbol into poolUpdate.Validate as the tokenSymbol. And then within poolUpdate.Validate we use the override if it exists.